lib/ueberauth/strategy/okta/oauth.ex

```diff
@@ -40,9 +40,17 @@ defmodule Ueberauth.Strategy.Okta.OAuth do
   Ueberauth.
   """
   def client(opts \\ []) do
-
-    config = :ueberauth
-             |> Application.fetch_env!(__MODULE__)
+    config = Application.fetch_env!(:ueberauth, __MODULE__)
+      @defaults
+      |> Keyword.merge(config)
+      |> Keyword.merge(opts)
+      |> validate_config_option!(:client_id)
+      |> validate_config_option!(:client_secret)
+      |> validate_config_option!(:site)
+      |> Client.new()
+      |> OAuth2.Client.put_serializer("application/json", Jason)
+
+    config = config
```

Comment:
If you use the implementation above, then you shouldn't need any of this below

Reply:
Yeah thanks!